e2e/client/playwright/page-object-models/media-editor.ts

```diff
@@ -0,0 +1,19 @@
+import {Locator, Page} from '@playwright/test';
+import {s} from '../utils';
+
+export class MediaEditor {
+    private page: Page;
+
+    constructor(page: Page) {
+        this.page = page;
+    }
+
+    field(field: string): Locator {
+        return this.page.locator(s('media-metadata-editor', field)).getByRole('textbox');
+    }
+
+    async saveMetadata(): Promise<void> {
+        await this.page.locator(s('media-editor', 'apply-metadata-button')).click();
+        await this.page.locator(s('change-image', 'done')).click();
```

Comment:
is `change-image` image selector still inside "media-editor"? If it is, it'd be good to scope it:
```
await this.page.locator(s('media-editor', 'change-image', 'done')).click();
```

The reason being not uniqueness, but documentation - ability to quickly understand the hierarchy without having to look it up inside dev tools.

Reply:
those are on the same level